src/modules/vmount/output_mavlink.cpp

```diff
@@ -104,7 +108,11 @@ int OutputMavlinkV1::update(const ControlData *control_data)
 	vehicle_command.param1 = math::degrees(_angle_outputs[1] + _config.pitch_offset);
 	vehicle_command.param2 = math::degrees(_angle_outputs[0] + _config.roll_offset);
 	vehicle_command.param3 = math::degrees(_angle_outputs[2] + _config.yaw_offset);
+#if defined(ALT_MANTIS_GIMBAL_HACKS)
+	vehicle_command.param7 = 1.0f; // This triggers angular rate control for the Mantis.
```

Comment:
If we set it to 2, then it's in absolute angle mode which means it will go back to horizontal whenever you let go of the wheel.

For the gimbal source, see PM.

Reply:
dislike (but I understand)